lib/network.go

```diff
@@ -2604,6 +2612,36 @@ func (msg *MsgBitCloutTxn) Sign(privKey *btcec.PrivateKey) (*btcec.Signature, er
 	return txnSignature, nil
 }
 
+// SignTransactionWithDerivedKey the signature contains solution iteration,
+// which allows us to recover signer public key from the signature.
+// Returns (new txn bytes, txn signature, error)
+func SignTransactionWithDerivedKey(txnBytes []byte, privateKey *btcec.PrivateKey) ([]byte, []byte, error){
```

Comment:
Sad, but I think we should delete this now.

Reply:
Ha! It is actually useful in the backend, https://github.com/AeonSw4n/backend/blob/fc59f/routes/admin_transaction.go#L351 , but I've updated the comment and method to reflect the new `ExtraData` approach.

Reply:
Ah ok looks good.